q2_metadata/plugin_setup.py

```diff
@@ -135,3 +136,31 @@ def _2(ff: ArtificialGroupingFormat) -> (qiime2.Metadata):
                  'values with sample ids will be random. These data will be '
                  'written to an artifact that can be used as sample metadata.')
 )
+
+
+plugin.methods.register_function(
+    function=merge,
+    inputs={},
+    parameters={'metadata1': Metadata,
+                'metadata2': Metadata},
+    parameter_descriptions={
+        'metadata1': 'First metadata file to merge.',
+        'metadata2': 'Second metadata file to merge.'
+    },
+    outputs=[('merged_metadata', ImmutableMetadata)],
+    output_descriptions={
+        'merged_metadata': 'The merged metadata.'
+    },
+    name='Merge metadata',
+    description=('Merge metadata that contains overlapping ids, overlapping '
```

Comment:
I'm not exactly sure how to reword this sentence, but it sounded a bit weird to me initially and I had to re-read a couple of times (just the 'neither overlapping ids nor overlapping columns' bit).

Reply:
Thanks for the input! I reworked this a bit to split the information into multiple sentences, and I think that should help. Let me know what you think.

Reply:
That reads much better, thanks @gregcaporaso! I'll go ahead and get this one merged.